src/app.ts

```diff
@@ -64,29 +69,64 @@ async function app() {
       'exit with error if this is not the latest version of SUSHI',
       false
     )
-    .option('-i, --init', 'initialize a SUSHI project')
-    .option('-u, --update-dependencies', 'update FHIR packages in project configuration')
-    .version(getVersion(), '-v, --version', 'print SUSHI version')
+    .action(async function (projectPath, options) {
+      await runBuild(projectPath, options);
+    })
     .on('--help', () => {
       console.log('');
       console.log('Additional information:');
       console.log('  [path-to-fsh-project]');
       console.log('    Default: "."');
       console.log('  -o, --out <out>');
       console.log('    Default: "fsh-generated"');
+    });
+
+  program
+    .command('updateDependencies [path-to-fsh-project]')
+    .description('update FHIR packages in project configuration')
+    .action(async function (projectPath) {
+      const input = ensureInputDir(projectPath);
+      const config: Configuration = readConfig(input);
+      await updateExternalDependencies(config);
+      process.exit(0);
     })
-    .arguments('[path-to-fsh-defs]')
-    .action(function (pathToFshDefs) {
-      input = pathToFshDefs;
-    })
-    .showHelpAfterError()
-    .parse(process.argv)
-    .opts();
+    .on('--help', () => {
+      console.log('');
+      console.log('Additional information:');
+      console.log('  [path-to-fsh-project]');
+      console.log('    Default: "."');
+    });
 
-  if (program.init) {
-    await init();
-    process.exit(0);
+  program
+    .command('init')
+    .description('initialize a SUSHI project')
+    .action(async function () {
+      await init();
+      process.exit(0);
+    });
+
+  // Maintain backwards compatibility with prior SUSHI command structure
+  if (
+    process.argv.length === 2 ||
+    (process.argv.length > 2 && !SUPPORTED_COMMANDS.includes(process.argv[2]))
+  ) {
+    if (process.argv.some(a => a === '-i' || a === '--init')) {
+      // init was moved to a separate command, so log a message to indicate how to use it
+      console.log(
+        'The --init option has been moved to a separate command. Instead, run the following command: sushi init'
+      );
+      process.exit(0);
```

Comment:
This should probably exit w/ an error code.
```suggestion
      process.exit(1);
```